src/Microsoft.DotNet.Build.CloudTestTasks/Microsoft.DotNet.Build.CloudTestTasks.csproj

```diff
@@ -26,22 +26,7 @@
     <Compile Include="$(CommonPath)\ZipFileCreateFromDirectory.cs">
       <Link>ZipFileCreateFromDirectory.cs</Link>
     </Compile>
-    <Compile Include="SendJobsToHelix.cs" />
-    <Compile Include="SimulateHelixExecution.cs" />
-    <Compile Include="ValidateWorkItems.cs" />
-    <Compile Include="AzureConnectionStringBuildTask.cs" />
-    <Compile Include="AzureHelper.cs" />
-    <Compile Include="CreateAzureContainer.cs" />
-    <Compile Include="DownloadFromAzure.cs" />
-    <Compile Include="FilterForUpload.cs" />
-    <Compile Include="GetPerfTestAssemblies.cs" />
-    <Compile Include="ListAzureContainers.cs" />
-    <Compile Include="SendToHelix.cs" />
-    <Compile Include="UploadClient.cs" />
-    <Compile Include="UploadToAzure.cs" />
-    <Compile Include="WriteBuildStatsJson.cs" />
-    <Compile Include="RemoveItemMetadata.cs" />
-    <Compile Include="WriteItemsToJson.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)*.cs" />
```

Comment:
I kept moving files over to buildtools then forgetting to add them to the csproj, so I changed this to a wildcard with the thought that msbuild based cli project systems do it this way anyways.  I can move this back to being explicit since there are pros-cons to both methods and I don't have a super strong need to make this change.